src/mono/mono/mini/driver.c

```diff
@@ -1843,6 +1847,9 @@ mono_jit_parse_options (int argc, char * argv[])
 		} else if (strncmp (argv [i], "--profile=", 10) == 0) {
 			mini_add_profiler_argument (argv [i] + 10);
 		} else if (argv [i][0] == '-' && argv [i][1] == '-' && mini_parse_debug_option (argv [i] + 2)) {
+#if HOST_BROWSER
+		} else if (argv [i][0] == '-' && argv [i][1] == '-' && mono_jiterp_parse_option (argv [i] + 2)) {
+#endif
```

Comment:
Shouldn't this be part of the --interp= options ? Also, there is a newer options api in utils/options.h which automatically makes options available both from the command line and C code.

Reply:
it's wasm specific, so it felt wrong to add it to the interp options. I can add it to those. The little typescript goo for querying them is also somewhat special and will get more complex/expensive if it queries ALL the interp settings instead of the jiterp ones, since the typescript part of the implementation needs to query the options to do the right things.

Reply:
do you have strong opinions on how the options interface for this should work? i.e. is options.h required, does it HAVE to be part of the --interp options, since parts of it interact with AOT and it has to interact with JS